.circleci/config.yml

```diff
@@ -56,3 +55,6 @@ workflows:
       - node/run:
           name: Check Spelling
           npm-run: spell:check
+      - node/run:
```

Comment:
Do we want to consider adding a pre-commit hook? Might be annoying to not notice build failures until late.

Reply:
Yeah, that's a nice to have. I'll be using it. Added it to our (now growing) set of opt-in repo git hooks [cdf70e1](https://github.com/apollographql/federation/pull/2785/commits/cdf70e1839e3cc1ff68a6ed4fa0217de5a65eae3).